cmd/ssl-expiry-check/README.md

```diff
@@ -0,0 +1,46 @@
+## SSL Expiration Status Check
+
+The *SSL Expiry Check* checks that SSL certificates are not currently expired, and that the expiration date is a specified number of days away (60 by default).
+
+The check runs daily (spec.runInterval), with a check timeout set to 5 minutes (spec.timeout). If the check
+does not complete within the given timeout it will report a timeout error on the status page.
+
+#### SSL Expiry Check Kube Spec:
+```yaml
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: ssl-expiry-check
```

Comment:
I think for the check names we're trying to avoid adding "check" at the end, so just `ssl-expiry` would be good!